src/common/kube-helpers.ts

```diff
@@ -4,6 +4,7 @@ import path from "path"
 import os from "os"
 import yaml from "js-yaml"
 import logger from "../main/logger";
+import { sync as commandExistsSync } from "command-exists"
```

Comment:
if we can use the `async` version I think that would be better since this is doing file system stuff

Reply:
@Nokel81 - The command-exists node module is usually imported with a require statement: `var commandExists = require('command-exists');`. However, our listing doesn't want us to use require statements so I created a type declaration for it but I am struggling with how to define the main function. It is declared as `module.exports = function commandExists(commandName, callback) {.....}` in the command-exists.js file - how does that get declared in my .d.ts file?

Reply:
@Nokel81 - Do we need to do this as we are looping over the configs 1 at a time and so we are only waiting for the 1 result to be returned.

I have tried to use the asynchronous and promise versions of command-exists but couldn't get them to work properly.

Reply:
Okay, if we aren't doing them in parallel then this is less of a concern.